### PR TITLE
awslogs: add formula

### PR DESCRIPTION
**Pull request by kevinburke**
Fixes jorgebastida/awslogs#116.

It would be nice if shell_output also captured stderr or if there
was a similar shell_stderr or shell_combined_output method that you
could use to capture that! I had to grep around to find a library that
tested in a similar manner.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Review by fxcoudert — APPROVED** [review]
The commit message should be `awslogs 0.7 (new formula)` but otherwise looks 👍

File: Formula/awslogs.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    sha256 "1d6d69ce66211143803fbc56652b41d73b4a400a2891d7bf7a1cdf4c02de613b"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    _, stderr, = Open3.capture3("#{bin}/awslogs --version")
+    assert "awslogs 0.7.0" == stderr.strip, "expected 'awslogs 0.7.0', got #{stderr}"
```
Comment (bfontaine) [inline Formula/awslogs.rb]: Use `assert_equal "awslogs 0.7.0", shell_output("#{bin}/awslogs --version 2>&1")`.

**Review by bfontaine — CHANGES_REQUESTED** [review]

File: Formula/awslogs.rb (hunk 1 above)
Comment (kevinburke) [inline Formula/awslogs.rb]: Huh, 2>&1 works inside of the string like that? OK...

Comment (kevinburke) [inline Formula/awslogs.rb]: Done

**Comment by kevinburke** [comment]
> The commit message should be awslogs 0.7 (new formula) but otherwise looks 👍

Done

**Review by bfontaine — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks a lot @kevinburke for your contribution to Homebrew!

**Pull request closed without merging**